### ISSUE TITLE
Radam not converging

### ISSUE THREAD

**Issue by ilyaskhanbey** [MarcoForte/FBA_Matting#46]
Hello, thnak you for your great work
I am trying to make the training code.
So i started by making alpha loss, composition loss and regression loss, and i used radam  optimizer 
**RAdam(group_weight(net, 1e-5, 1e-5, 0.0001))** as you mentionned.
To test the convegenve and evrithing is right in my code i took only 2 images with hair from adobe dataset and start training with these 2 images.
But it's seems the loss is  not converging, however when using the classical adam optimizer i got convergence in few iterations.
the **group_weight** function seems dealing with parameters of the resnet encoder, i think  i should use a pretrained resnet.
could you share the pretrained  resnet you used and if possible the code to load it in matting module.
thanks in advance

**Comment by SysDevHayes** [comment]
@ilyaskhanbey I think I have encountered the same issue. Even after successfully loading the pretrained resnet. Have you solved the issue?
